provisioning/device/src/CertificateInstaller.cs

```diff
@@ -1,68 +1,31 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using Microsoft.Azure.Devices.Shared;
-using System;
-using System.Collections.Generic;
 using System.Security.Cryptography.X509Certificates;
+using Microsoft.Azure.Devices.Shared;
 
 namespace Microsoft.Azure.Devices.Provisioning.Client
 {
     internal static class CertificateInstaller
     {
-        private static readonly HashSet<string> s_installedCertificates = new HashSet<string>();
```

Comment:
By saving all the thumbprints installed in the user's cert store in a static member of a static class, it will never be released for the duration of the program.

At first, I thought about making the class an instance class so the thumbprint list could be used over and over, but then I looked at usage and we really just use it once, on construction of the DPS client when using a cert.

So I figured we might as well do less with this code. It shouldn't be any less performant.